src/stpipe/step.py

```diff
@@ -356,6 +359,17 @@ def _check_args(self, args, discouraged_types, msg):
                     "{0} {1} object.  Use an instance of AbstractDataModel instead.".format(
                         msg, i))
 
+    @property
+    def log_records(self):
```

Comment:
Is this a good name for this method, like `get_records` or `retrieve_records`?  This method name sounds like it's logging records, but the function returns records.

Reply:
Since it's a property it should be a noun, I think, but I'm open to other suggestions.  I didn't notice the similarity between "log_records" and "record_logs" until now...